api-docs/autogen/api/v2_groups_group_id_upload_prepare.post.md

```
@@ -0,0 +1,164 @@
+
+## [#](#请求) 请求
+
+### [#](#基础信息) 基础信息
+
+|字段|值|
+|---|---|
+|HTTP URL|/v2/groups/{group\_id}/upload\_prepare|
+|HTTP Method|POST|
+|接口频率限制|10 QPS|
+
+## [#](#路径参数) 路径参数
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)